README.md

```diff
@@ -12,8 +12,7 @@ There are 4 main components in the header package:
  1. p2p.Subscriber listens for new Headers from the P2P network (via the
     HeaderSub)
  2. p2p.Exchange request Headers from other nodes
- 3. Syncer manages syncing of past and recent Headers from either the P2P
-    network
+ 3. Syncer manages syncing of past and recent Headers from the P2P network
```

Comment:
note: past and recent headers seem similar. Maybe this line was trying to convey past (like distant past) and recent (like recently past) headers?

Perhaps this could read

```suggestion
 3. Syncer manages syncing of historical Headers from the P2P network
```